tools/packaging/rpm/Dockerfile.build

```diff
@@ -1,22 +1,36 @@
 FROM centos:7
 
-RUN yum upgrade -y && \
-    yum install -y epel-release centos-release-scl && \
-    yum install -y fedpkg sudo make which cmake3 rh-git218 \
-                   automake autoconf autogen libtool ninja-build \
-                   llvm-toolset-7 devtoolset-7-libatomic-devel && \
+RUN yum install -y centos-release-scl epel-release && \
+    yum update -y && \
+    yum install -y fedpkg sudo devtoolset-7-gcc devtoolset-7-gcc-c++ \
+                   devtoolset-7-binutils java-1.8.0-openjdk-headless rsync \
+                   rh-git218 wget unzip which make cmake3 patch ninja-build \
+                   devtoolset-7-libatomic-devel openssl python27 libtool autoconf && \
     yum clean all
 
 # Install go (go package in yum repositories is too old)
-RUN curl -o /root/go.tar.gz https://dl.google.com/go/go1.12.7.linux-amd64.tar.gz && \
+RUN curl -o /root/go.tar.gz https://dl.google.com/go/go1.13.4.linux-amd64.tar.gz && \
     tar zxf /root/go.tar.gz -C /usr/local
 ENV GOROOT=/usr/local/go \
-    PATH=/usr/local/go/bin:${PATH}
+    PATH=/usr/local/go/bin:/opt/rh/rh-git218/root/usr/bin:/opt/rh/devtoolset-7/root/usr/bin:/opt/llvm/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:${PATH}
 
 # Install bazel
-RUN curl -o /root/bazel-installer.sh -L http://github.com/bazelbuild/bazel/releases/download/0.27.1/bazel-0.27.1-installer-linux-x86_64.sh && \
-    chmod +x /root/bazel-installer.sh && \
-    /root/bazel-installer.sh
+RUN curl -o /usr/local/bin/bazel -L https://github.com/bazelbuild/bazelisk/releases/download/v0.0.8/bazelisk-linux-amd64 && \
+    chmod +x /usr/local/bin/bazel
```

Comment:
Hmm, didn't know about `bazelisk`. Is it safe/stable to use?

Reply:
This is the recommendation on Envoy for installing bazel -
https://github.com/envoyproxy/envoy/blob/master/bazel/README.md#installing-bazelisk-as-bazel

Reply:
ok, no objections from me, if it works fine. Does it make sense to use the latest one (1.1.0)?

Reply:
I have updated to the latest one - 1.1.0